README.md

```diff
@@ -4,6 +4,8 @@
   <img src="assets/goose_logo.png" width="125"">
 </p>
 
+*[The Go gopher](https://go.dev/blog/gopher) was designed by [Renée French](https://reneefrench.blogspot.com/).*
```

Comment:
I don't think this should go at the very top of the Readme. Probably best to place it above License header, maybe Credits?

The original Gopher mascot was indeed created by Renée, but this gopher is an adaptation by a friend, Ellen.

Here is the wording I have used elsewhere, what do you think?

> The gopher mascot was designed by [Renée French](https://reneefrench.blogspot.com/) / [CC 3.0](https://creativecommons.org/licenses/by/3.0/). For more info check out the [Go Blog](https://blog.golang.org/gopher). Adapted by

Reply:
Thanks for your comment! Agree. I just placed it near the image but your idea seems better than mine. I fixed it with this commit https://github.com/pressly/goose/pull/375/commits/97b3e4c0a022becd6cee8dec7231f16bd8992e94 . Please let me know if Ellen has her own website, I'll add it if needed.